### ISSUE TITLE
mongodb causes make build to fail

### ISSUE THREAD

**Issue by 0xffea** [uselagoon/lagoon#179]
```
$ make build
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/centos7 -f images/centos7/Dockerfile images/centos7
sha256:2159237d71f5f41600a4deb6a8e588a249b5e78bdb9a04035e9a5885eb5591e2
touch build/centos7
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/centos7-mariadb10 -f images/centos7-mariadb10/Dockerfile images/centos7-mariadb10
sha256:925af22f0679d4ca51fd68b6fa9efd2d6dbbdd0e7cc9087eba65d362a98d5f52
touch build/centos7-mariadb10
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/centos7-mariadb10-drupal -f images/centos7-mariadb10-drupal/Dockerfile images/centos7-mariadb10-drupal
sha256:3104e7579db13094f3b44549e528b2ccb5524feec5386848a93397d066d83bf3
touch build/centos7-mariadb10-drupal
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/oc-build-deploy-dind -f images/oc-build-deploy-dind/Dockerfile images/oc-build-deploy-dind
sha256:ddb666a0248a193d57ddc911f0f022f5f9e544eb96b4b7067bb4d7e991e9b1d6
touch build/oc-build-deploy-dind
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/nginx -f images/nginx/Dockerfile images/nginx
sha256:fcbe950941106886afa6aa86e2b45b4bfd0cb64acb248e67cd7d83887373bd05
touch build/nginx
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/nginx-drupal -f images/nginx-drupal/Dockerfile images/nginx-drupal
sha256:27ad8d693fc8bb923ddb9bf0bfac0160c39e3b75f31131db2859ea71fdb6afb5
touch build/nginx-drupal
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/varnish -f images/varnish/Dockerfile images/varnish
sha256:1bc10f5661e6fb60121c38e5a6acbeb5b9a0d3a73a3b82cacaf9aae7a94416e2
touch build/varnish
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/varnish-drupal -f images/varnish-drupal/Dockerfile images/varnish-drupal
sha256:72977b1585631fb7d59e46f6835c24e701144743db8edb5bb62f02c1d1683b2f
touch build/varnish-drupal
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/redis -f images/redis/Dockerfile images/redis
sha256:a7096cce92fd4422e2029dece4517137dc79179e6051d9ac7b884f735d8839a5
touch build/redis
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/mongo -f images/mongo/Dockerfile images/mongo
Sending build context to Docker daemon   2.56kB
Step 1/13 : ARG IMAGE_REPO
 ---> 
Step 2/13 : FROM ${IMAGE_REPO:-lagoon}/commons as commons
 ---> 1697510961c7
Step 3/13 : FROM alpine
 ---> baa5d63471ea
Step 4/13 : MAINTAINER amazee.io
 ---> Using cache
 ---> 67c290d1e3b4
Step 5/13 : ENV LAGOON mongo
 ---> Using cache
 ---> 33389ec8ea9f
Step 6/13 : COPY --from=commons /lagoon /lagoon
 ---> Using cache
 ---> 81425de032cc
Step 7/13 : COPY --from=commons /bin/fix-permissions /bin/ep /bin/docker-sleep /bin/
 ---> Using cache
 ---> c7f47817d5c3
Step 8/13 : COPY --from=commons /home /home
 ---> Using cache
 ---> 48457db9e33b
Step 9/13 : ENV TMPDIR /tmp TMP /tmp HOME /home ENV /home/.bashrc BASH_ENV /home/.bashrc
 ---> Using cache
 ---> a4dfda25172d
Step 10/13 : RUN echo "@edge http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories   && apk --no-cache add mongodb@edge
 ---> Running in 13a89efd890c
fetch http://dl-cdn.alpinelinux.org/alpine/v3.4/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.4/community/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
WARNING: This apk-tools is OLD! Some packages might not function properly.
ERROR: unsatisfiable constraints:
  mongodb (missing):
    required by: world[mongodb]
The command '/bin/sh -c echo "@edge http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories   && apk --no-cache add mongodb@edge' returned a non-zero code: 1
Makefile:123: recipe for target 'build/mongo' failed
make: *** [build/mongo] Error 1
```

**Comment by Schnitzel** [comment]
@twardnw can you check that?

**Comment by twardnw** [comment]
huh, appears that mongodb is no longer in the edge repo, or the main repo. Will investigate.

**Comment by Schnitzel** [comment]
fixed in #183
